### PR TITLE
[Packaging] update the debian packaging

### PR DESCRIPTION
**Pull request by helloahn**
update rules and install path

Signed-off-by: Hyoung Joo Ahn <hello.ahn@samsung.com>

### CODE REVIEW

File: packaging/protobuf.spec [hunk 1]
```diff
@@ -5,7 +5,9 @@ Release:        1%{?dist}
 License:        BSD
 Group:          Development/Libraries
 Source:         https://github.com/google/protobuf/releases/download/v%{version}/protobuf-all-%{version}.tar.gz
-Source1001:     protobuf.manifest
+#Source1:        proto.vim
+#Patch3:         0001-Add-generic-GCC-support-for-atomic-operations.patch
+#Patch4:         protobuf-2.5.0-makefile.patch
```
Comment (myungjoo) [inline packaging/protobuf.spec:10]: Is it safe to remove manifest for Tizen systems? It usually makes troubles in run time.

File: packaging/protobuf.spec [hunk 2]
```diff
@@ -133,51 +133,42 @@ install -p -m 644 -D editors/proto.vim %{buildroot}%{_datadir}/vim/vimfiles/synt
...
 %files lite-static
-%manifest protobuf.manifest
 %defattr(-, root, root, -)
 %{_libdir}/libprotobuf-lite.a
 
 %files vim
-%manifest protobuf.manifest
 %defattr(-, root, root, -)
-%{_datadir}/vim/vimfiles/syntax/proto.vim
+%{_datadir}/vim/vimfiles/syntax/proto.vim
```
Comment (myungjoo) [inline packaging/protobuf.spec:174]: Check your editor.

File: debian/control [hunk 3]
```diff
@@ -1,10 +1,18 @@
...
+ , debhelper (>= 9)
+ , autotools-dev
+ , dh-autoreconf
+# C/C++
+ , zlib1g-dev
+ , libgtest-dev
 Maintainer: Laszlo Boszormenyi (GCS) <gcs@debian.org>
 Standards-Version: 3.6.1
 Homepage: https://github.com/protocolbuffers/protobuf
+#Vcs-Git: git://github.com/protocolbuffers/protobuf
```
Comment (wooksong) [inline debian/control:15]: I do not see why this line is needed.

**Comment by helloahn** [comment]
@wooksong will figure it out.

**Pull request closed without merging**